imagersite/imager_images/models.py

```diff
@@ -0,0 +1,55 @@
+from django.db import models
+from django.conf import settings
+from django.utils.encoding import python_2_unicode_compatible
+
+
+PUBLISHED_CHOICES = (('private', 'private'), ('public', 'public'), ('shared', 'shared'))
+PUBLISHED_DEFAULT = PUBLISHED_CHOICES[0][1]
+
+
+@python_2_unicode_compatible
+class Photo(models.Model):
+    """Photo class."""
+
+    image = models.ImageField(upload_to='media')
+    title = models.CharField(max_length=250)
+    description = models.TextField()
+    uploaded = models.DateTimeField(auto_now_add=True)
+    modified = models.DateTimeField(auto_now=True)
+    album = models.ManyToManyField('Album', related_name='album')
```

Comment:
For these M2M relationships between Photos and Albums:
 - You should only have the relationship defined on one of the models; you don't need both the `album` field on `Photo` **and** the `photo` field on `Album`. When you do this, you're actually creating two completely separate relationships.
 - The string name of the model to which you're relating should be in the format `'app_name.model_name'`, not just `'model_name'`.